ui/app/helpers/mountable-secret-engines.js

```diff
@@ -1,84 +1,90 @@
-import { helper as buildHelper } from '@ember/component/helper';
+import { helper as buildHelper } from "@ember/component/helper";
```

Comment:
I think we're doing single quote for JS files - that's in the .editorconfig file - maybe we need an explicit prettierconfig / prettierrc file? Ohh or have eslint run prettier - I think that's probably what we need to do. Sorry for the nit, we just need to get it automated and then run prettier/eslint on the whole codebase.